assets/css/governance.scss

```diff
@@ -0,0 +1,19 @@
+.circle-profile-img {
+  border-radius: 50%;
+  -webkit-box-shadow: none;
+  box-shadow: none;
+  overflow: hidden;
+}
+
+.person-card {
+    text-align: center;
+    padding: var(--oe-padding-small);
+}
+
+.person-card sl-details {
+   font-size: 15px;
```

Comment:
i always avoid pixel measurements, they don't scale as well on zoom

try percentage ( e.g. `0.75%`) or a fraction of em (`0.75 em`)